gluten-ut/spark35/src/test/scala/org/apache/spark/sql/execution/GlutenStreamingQuerySuite.scala

```diff
@@ -0,0 +1,29 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.spark.sql.execution
+
+import org.apache.gluten.config.GlutenConfig
+
+import org.apache.spark.SparkConf
+import org.apache.spark.sql.GlutenSQLTestsTrait
+import org.apache.spark.sql.streaming._
+
+class GlutenStreamingQuerySuite extends StreamingQuerySuite with GlutenSQLTestsTrait {
+  override def sparkConf: SparkConf =
+    super.sparkConf
+      .set(GlutenConfig.RAS_ENABLED.key, "true")
```

Comment:
We had this CI job https://github.com/apache/incubator-gluten/blob/aa485e53767d7ebac12477314693577ebc683e88/.github/workflows/velox_backend_x86.yml#L1051 running Spark UTs with `RAS=ON` already. So probably we don't have to explicitly set this option to test RAS.